src/services/CacheService/index.ts

```diff
@@ -0,0 +1,78 @@
+import { ONE_DAY_TIMESTAMP } from '../../constants/common'
+import CONFIG from '../../config'
+import { PersistenceService, persistenceService } from '../PersistenceService'
+
+export const NAMESPACE_CACHE = 'cache'
+
+export interface CacheOptions {
+  expire?: number
+  // When expire is present, ignore expire.
```

Comment:
This comment is confusing, and they may be presented in one key such as

```js
expireAt: Date | Enum
```
`Date` for a specific time, and `Enum` for a duration since now.

Reply:
I wrote it wrong, it should be `When expireAt is present, ignore expire.`

What is an `Enum`? Are you saying that `expireAt` has the ability to handle both `date` and `duration`?

Reply:
Yes, `expireAt` could be `date` or `duration`, and `Date` for a specific time, `Enum` for a duration

Enum could be
```ts
Enum Duration {
  OneHour,
  OneDay,
  OneWeek,
  OneMonth,
  OneYear,
}
```

Reply:
If an `Enum` is used, it may be a bit redundant, and most of the time special durations will have to be converted to `Date` to be used.

If it is `Date | number`, it is easy to misunderstand the purpose. Therefore, I thought about it and it's better to have two properties.

The meaning of `expireAt` is more obvious, and `expire` can be renamed to `expireTime` or `duration`.

For example:
```typescript
type MutexAttr<T1 extends Object, T2 extends Object> =
  | ({
      [K1 in keyof T1]: T1[K1]
    } & {
      [K2 in keyof T2]?: never
    })
  | ({
      [K1 in keyof T1]?: never
    } & {
      [K2 in keyof T2]: T2[K2]
    })

type ExpireOptions = MutexAttr<{ expireTime?: number }, { expireAt?: number }>
```

Reply:
done